### PR TITLE
6013 skynet page structure

### PR DESCRIPTION
**Pull request by im-ayakovtsov**
added switching by tabs on pages

### CODE REVIEW

File: cypress/src/skynet/pages/myAccountPage.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  clickChangePassword() {
+    return cy.get(this.changePasswordButton)
+      .click()
+  }
+
+  clickSaveButton() {
+    return cy.get(this.saveButton)
+      .should('not.be.disabled')
```
Comment (im-obondaruk) [inline cypress/src/skynet/pages/myAccountPage.js:41]: No assertions in the pages.

File: cypress/src/skynet/po.js [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
   }
 }
 
 const fragments = {
   applicationHeader: {
-    applicationHeader: new HeaderFragment()
+    header: new HeaderFragment()
   },
 
   navigationBar: {
```
Comment (im-obondaruk) [inline cypress/src/skynet/po.js:22]: Maybe we could call the fragment sideNavBar, and let it be on the first level of fragments' object properties?

File: cypress/src/skynet/po.js [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
     navbar: new NavbarFragment()
+  },
+
+  formFragment: {
```
Comment (im-obondaruk) [inline cypress/src/skynet/po.js:26]: Let's discuss this level of fragments

File: cypress/src/skynet/sharedFragments/formFragment.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+  enterConfirmPassword(value) {
+    return cy.get(this.confirmPasswordField)
+      .clear()
+      .type(value);
+  }
+
+  clickSaveButton() {
+    return cy.get(this.saveButton)
+      .should('not.be.disabled')
```
Comment (im-obondaruk) [inline cypress/src/skynet/sharedFragments/formFragment.js:39]: No assertion in the page class

File: cypress/src/skynet/sharedFragments/navbarFragment.js [hunk 5]
```diff
@@ -34,6 +34,13 @@ class NavbarFragment {
     return cy.get(this.companyItem).click();
   }
 
+  openCompanyList(value) {
+    //open the page from the drop-down list of the Companies menu
+    this.openCompany()
+    return cy.get(`[data-test=${value}]`)
```
Comment (im-obondaruk) [inline cypress/src/skynet/sharedFragments/navbarFragment.js:40]: locator should be in the constructor. Make it a function that returns a string depend on what value you've passed.
`this.companyElem = (companyValue) => "[data-test=${value}]"`

File: cypress/src/skynet/sharedFragments/navbarFragment.js [hunk 6]
```diff
@@ -42,6 +49,13 @@ class NavbarFragment {
     return cy.get(this.systemsItem).click();
   }
 
+  openSystemList(value) {
+    //open the page from the drop-down list of the Systems menu
+    this.openSystem()
+    return cy.get(`[data-test=${value}]`)
```
Comment (im-obondaruk) [inline cypress/src/skynet/sharedFragments/navbarFragment.js:55]: locator should be as a function in constructor.

**Review by im-obondaruk — CHANGES_REQUESTED** [review]

**Comment by im-ayakovtsov** [comment]
Will be improved in the new PR

**Pull request closed without merging**